src/cmd/singularity/inspect_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package main
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/sylabs/singularity/src/pkg/test"
+)
+
+func inspectImage(image string) ([]byte, error) {
+	args := []string{"inspect", image}
+	cmd := exec.Command(cmdPath, args...)
+
+	return cmd.CombinedOutput()
+}
+
+func TestInspect(t *testing.T) {
+	var definition = "../../../examples/busybox/Singularity"
+	var image = "Default.sif"
+
+	if b, err := buildImage(definition, image); err != nil {
```

Comment:
I think we could add a labels section to that busybox definition
```
%labels

    Maintainer sylabs

    Version v3.0.0-beta
```

Reply:
well, I think there should already be some labels there by default right?  Like a timestamp of when the container was created.  So I don't know if we need to add any new ones.